src/Samples/DIPS.Xamarin.UI.Samples/Controls/Toast/ToastPage.xaml.cs

```diff
@@ -0,0 +1,142 @@
+using System.Threading.Tasks;
+using System.Windows.Input;
+using DIPS.Xamarin.UI.Controls.Toast;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+using Color = System.Drawing.Color;
+using ToastControl = DIPS.Xamarin.UI.Controls.Toast.Toast;
+
+namespace DIPS.Xamarin.UI.Samples.Controls.Toast
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class ToastPage : ContentPage
+    {
+        private string m_pageTitle;
+        private ICommand m_showToastCommand;
+        private ICommand m_showToastCommand2;
+        private ICommand m_showToastCommand3;
+
+        public ToastPage()
+        {
+            InitializeComponent();
+
+            PageTitle = "Hello, World!";
+
+            ShowToastCommand = new Command(parameter =>
+            {
+                ToastControl.Current.ToastLayout = LayoutOneMoonFourMars;
+                ToastControl.Current.HideToastIn = 5;
+                ToastControl.Current.ToastAction = async () =>
+                {
+                    PageTitle = "Hello, Mercury!";
+                    await Task.Delay(1500);
+                    PageTitle = "Hello, World!";
+                };
+                ToastControl.Current.Text = parameter.ToString();
+
+                ToastControl.Current.DisplayToast();
+            });
+
+            ShowToastCommand2 = new Command(parameter =>
+            {
+                ToastControl.Current.ToastLayout = LayoutTwoVenus;
+                ToastControl.Current.HideToastIn = 0;
+                ToastControl.Current.ToastAction = null;
+                ToastControl.Current.Text = parameter.ToString();
+
+                ToastControl.Current.DisplayToast();
+            });
+
+            ShowToastCommand3 = new Command(parameter =>
+            {
+                ToastControl.Current.ToastLayout = LayoutThreePluto;
+                ToastControl.Current.HideToastIn = 5;
+                ToastControl.Current.ToastAction = async () =>
+                {
+                    await ToastControl.Current.CloseToast();
+                };
+                ToastControl.Current.Text = parameter.ToString();
+
+                ToastControl.Current.DisplayToast();
+            });
+        }
+
+        public string PageTitle
+        {
+            get => m_pageTitle;
+            set
+            {
+                m_pageTitle = value;
+                OnPropertyChanged(nameof(PageTitle));
+            }
+        }
+
+        public ICommand ShowToastCommand
+        {
```

Comment:
Remove backing fields for commands to keep examples short and clean. This can be a getter and setting it in the ctor

Reply:
backing fields had to be included to trigger the property change event to make the binding work.

I'll use a ViewModel for the sample which will in return remove the need to invoke the call from the code behind